### PR TITLE
Main menu

### PR DESCRIPTION
**Pull request by manuelkoso**

### CODE REVIEW

**Comment by manuelkoso** [comment]
In my opinion, the `toString` method of the `FreedomLine` class is ok.

File: core/src/it/units/sdm/project/game/gui/MainMenuScreen.java [hunk 1]
```diff
@@ -2,23 +2,44 @@
...
+        skin = new Skin(Gdx.files.internal("skin/uiskin.json"));
+        initialMenu = new Table();
+        initMenu();
+        stage.addActor(initialMenu);
+        Gdx.input.setInputProcessor(stage);
+    }
+
+    private void initMenu() {
+        initialMenu.setFillParent(true);
+        Image image = new Image(new Texture(Gdx.files.internal("freedom_logo.png")));
```
Comment (peiva-git) [inline core/src/it/units/sdm/project/game/gui/MainMenuScreen.java:36]: We should dispose of textures that aren't used anymore. We'll have to generate an appropriate texture pack to use with an atlas at some point, but for now we should at least dispose of them in the `dispose()` method

File: core/src/it/units/sdm/project/game/gui/MainMenuScreen.java [hunk 2]
```diff
@@ -2,23 +2,44 @@
...
+        Label nameLabel = new Label("Tap anywhere to begin!", skin);
+        nameLabel.setColor(Color.BLACK);
+        nameLabel.setFontScale((float) 1.5);
```
Comment (peiva-git) [inline core/src/it/units/sdm/project/game/gui/MainMenuScreen.java:39]: I refactored this to `1.5f`

**Review by peiva-git — APPROVED** [review]
Looks neat!

**Pull request merged**